prboom2/src/r_defs.h

```diff
@@ -351,6 +353,12 @@ typedef struct line_s
   int healthgroup;
   const byte* tranmap;
   float alpha;
+
+  // gl_preprocess
+  dboolean tess:1;
```

Comment:
Please reuse the `r_flags` field if we're adding more temporary flags like this. You'll have to make `r_flags_t` an `unsigned short` instead of a `byte` in that case.